solara/server/settings.py

```diff
@@ -132,6 +132,7 @@ class Config:
 
 class Session(BaseSettings):
     secret_key: str = SESSION_SECRET_KEY_DEFAULT
+    http_only: Optional[bool] = None
```

Comment:
```suggestion
    http_only: bool = False
```

I think we can do it like this. The reason https_only is 'tri-state' is that we do some checks when it is not explicitly set.